src/services/coreBackService.test.js

```diff
@@ -54,6 +58,8 @@ describe("CoreBackService", () => {
           "feature-set": "test_feature_set",
           "ipv-session-id": "test_ipv_session_id",
           "client-session-id": "test_client_session_id",
+          "txma-audit-encoded": "dummy-txma-header",
+          "x-forwarded-for": "127.0.0.1",
```

Comment:
I think it's risky to hardcoded an ip here. Maybe the value will not always be this one.
What you can do is to set a value line 43 and reuse it here.

Same comment applies for others line which defines an ip.